entrypoint.sh

```diff
@@ -39,6 +39,16 @@ if [[ $LOG_LEVEL ]]; then
     sed -i -e"s/^.*log_level:.*$/log_level: ${LOG_LEVEL}/" /etc/dd-agent/datadog.conf
 fi
 
+if [[ $DD_LOGS_STDOUT ]]; then
+  export LOGS_STDOUT=$DD_LOGS_STDOUT
+fi
+
+if [[ $LOGS_STDOUT == "yes" ]]; then
+  sed -i -e "/^.*_logfile.*$/d" /etc/dd-agent/supervisor.conf
+  sed -i -e "/^.*\[program:.*\].*$/a stdout_logfile=\/dev\/stdout\nstdout_logfile_maxsize=0\nstderr_logfile=\/dev\/stderr\nstderr_logfile_maxsize=0" /etc/dd-agent/supervisor.conf
+  sed -i -e "$ a disable_file_logging: yes" /etc/dd-agent/datadog.conf
```

Comment:
I don't think we should disable file logging. It would make the flare command pointless and users may want to use `docker logs` and still be able to ask for support.